src/components/TaskPopup.jsx

```diff
@@ -69,17 +69,25 @@ const TaskPopup = ({ task, onClose }) => {
           <div className={styles.ExtraBtnContainer}>
             <div className={styles.Extra}>
               <h4 className={styles.ExtraTitle}>Add to card</h4>
-              <button className={styles.ExtraBtn}>Deadline</button>
-              <button className={styles.ExtraBtn}>Do Date</button>
-              <button className={styles.ExtraBtn}>Member</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
-              <button className={styles.ExtraBtn}>Extra</button>
+              <button className={`${styles.ExtraBtn} ${styles.DeadlineBtn}`}>
+                Deadline
+              </button>
```

Comment:
This button is wrapped in a div, and makes the styling on mobile look weird.

![image](https://github.com/EllenSandqvist/kanban_collab/assets/58448612/ef03835f-acd3-4c0f-8790-2e44de3b2dcf)